ext/sqlite3/sqlite3.c

```diff
@@ -1386,14 +1387,14 @@ PHP_METHOD(sqlite3stmt, readOnly)
 	SQLITE3_CHECK_INITIALIZED(stmt_obj->db_obj, stmt_obj->initialised, SQLite3);
 	SQLITE3_CHECK_INITIALIZED_STMT(stmt_obj->stmt, SQLite3Stmt);
 
-#if SQLITE_VERSION_NUMBER >= 3007004
 	if (sqlite3_stmt_readonly(stmt_obj->stmt)) {
 		RETURN_TRUE;
 	}
-#endif
+
 	RETURN_FALSE;
 }
 /* }}} */
+#endif
```

Comment:
You'll also have to ifdef the corresponding PHP_ME entry.

Reply:
Oh right missed that, done, thanks!